llvm/lib/Transforms/Vectorize/LoopVectorizationPlanner.h

```diff
@@ -276,6 +276,20 @@ class VPBuilder {
     return tryInsertInstruction(new VPPhi(IncomingValues, DL, Name));
   }
 
+  VPValue *createElementCount(Type *Ty, ElementCount EC) {
+    VPlan &Plan = *getInsertBlock()->getPlan();
+    VPValue *RuntimeEC =
+        Plan.getOrAddLiveIn(ConstantInt::get(Ty, EC.getKnownMinValue()));
+    if (EC.isScalable()) {
+      VPValue *VScale = createNaryOp(VPInstruction::VScale, {}, Ty);
+      RuntimeEC = EC.getKnownMinValue() == 1
+                      ? VScale
+                      : createNaryOp(Instruction::Mul, {VScale, RuntimeEC},
+                                     VPIRFlags::WrapFlagsTy(true, false));
```

Comment:
Nit, can you use createOverflowingOp here to avoid the WrapFlagsTy constructor? Not sure if it's worth it or not but we seem to have the helper anyway.